traffic_ops/app/bin/checks/DnssecRefresh/ToDnssecRefresh.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/apache/trafficcontrol/lib/go-log"
+	"github.com/apache/trafficcontrol/traffic_ops/app/bin/checks/DnssecRefresh/config"
+	"io/ioutil"
+	"net/http"
+	"net/http/cookiejar"
+	"os"
+)
+
+func main() {
+	cfg, err := config.GetCfg()
+	config.ErrCheck(err)
+	log.Debugln("Including DEBUG messages in output. Config is:")
+	config.PrintConfig(cfg) // only if DEBUG logging is set.
+	body := &config.Creds{
+		User:     cfg.TOUser,
+		Password: cfg.TOPass,
+	}
+	loginUrl := fmt.Sprintf("%s%s", cfg.TOUrl, "/api/2.0/user/login")
+	buf := new(bytes.Buffer)
+	err = json.NewEncoder(buf).Encode(body)
+	config.ErrCheck(err)
+	req, _ := http.NewRequest("POST", loginUrl, buf)
+	jar, _ := cookiejar.New(nil)
+	client := &http.Client{Jar: jar, Transport: cfg.Transport}
+
+	log.Debugf("Posting to: %s", loginUrl)
+
+	res, err := client.Do(req)
+	config.ErrCheck(err)
+	defer config.Dclose(res.Body)
+	refreshUrl := fmt.Sprintf("%s%s", cfg.TOUrl, "/api/2.0/cdns/dnsseckeys/refresh")
+	resp, _ := http.NewRequest("GET", refreshUrl, buf)
+
+	log.Debugf("Get req to: %s", refreshUrl)
+
+	refresh, _ := client.Do(resp)
+	respData, _ := ioutil.ReadAll(refresh.Body)
+	defer config.Dclose(refresh.Body)
```

Comment:
Not sure what the purpose of `Dclose` is? This could just be `defer refresh.Body.Close()`

Reply:
`Dclose` was a func I came up with to satisfy goland. It complained about unhandled errors with just `defer refresh.Body.Close()`, and I couldn't make it go away. Since I use it more than once I put it in a func so didn't have to type `if err != nil {}`. I guess it kind of my Nitpic. If you think it troublesome I'm happy to remove it. I did make it log and error now if called. seem to have left that out.